addon/bootstrap.js

```diff
@@ -31,9 +31,11 @@ this.Bootstrap = {
   STUDY_DURATION_WEEKS: 2,
 
   /**
-   * @param addonData Array [ "id", "version", "installPath", "resourceURI", "instanceID", "webExtension" ]  bootstrap.js:48
-   * @param reason
-   * @returns {Promise<void>}
+   * TODO
+   *
+   * @param   {Object} addonData [ "id", "version", "installPath", "resourceURI", "instanceID", "webExtension" ]  bootstrap.js:48
```

Comment:
The old code comment said this was an Array, and gave these values, but it _felt_ like an Object to me, since we're trying to access `addonData.id` and `addonData.version` below. ¯\\\_(ツ)\_/¯